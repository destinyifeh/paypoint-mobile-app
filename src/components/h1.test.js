'use strict';

import React from 'react';
import H1 from './h1';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<H1 />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it('underlined h1 remains the same', () => {
  const tree = renderer
    .create(<H1
      underline
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
