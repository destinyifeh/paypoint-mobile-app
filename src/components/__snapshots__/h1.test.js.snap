// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#00425F",
        "fontFamily": "Source Sans Pro",
        "fontSize": 22,
        "fontWeight": "bold",
      }
    }
  />
</View>
`;

exports[`underlined h1 remains the same 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#00425F",
        "fontFamily": "Source Sans Pro",
        "fontSize": 22,
        "fontWeight": "bold",
      }
    }
  />
  <View
    style={
      Object {
        "alignSelf": "center",
        "backgroundColor": "#bcbbc1",
        "height": 0.5,
        "marginBottom": 20,
        "width": 100,
      }
    }
  />
</View>
`;
